common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -247,97 +232,63 @@ nav.sequence-nav {
     }
   }
 
-  ul {
-    position: absolute;
-    top: 0;
-    list-style: none !important;
-    height: 100%;
-    right: 0;
-    top: 0;
-    width: 100%;
-    margin: 0;
-    border: none;
-
-    li {
-      position: absolute;
-      margin-bottom: 0;
-      height: 44px;
-      width: 70px;
-      border: 1px solid $gray-l3;
-      @include linear-gradient(top, #eee, #ddd);
-      box-shadow: 0 1px 0 rgba(255, 255, 255, .7) inset;
-      z-index: 1;
-
-      &.prev, &.next {
-
-        a {
-          background-position: center;
-          background-repeat: no-repeat;
-          display: block;
-          height: 100%;
-          width: 40px;
-          text-indent: -9999px;
-          overflow: hidden;
-          @include transition(all .2s $ease-in-out-quad 0s);
-
-/*          &:focus {
-            outline: 0;
-          }
-*/
-          &:hover, &:focus {
-            opacity: 0.5;
-          }
-
-          &.disabled {
-            cursor: normal;
-            opacity: 0.4;
-          }
-        }
-      }
+  body.touch-based-device & ol li a:hover p {
+    display: none;
+  }
+}
 
-      &.prev {
-        @include left(-10px);
-        @include border-radius(35px, 0, 0, 35px);
+.sequence-nav-button {
+  @extend %ui-depth3;
+  position: absolute;
+  display: block;
+  top: 0;
+  width: ($baseline*2);
+  height: 46px;
+  background-position: center;
+  background-repeat: no-repeat;
+  text-indent: -9999px;
+  overflow: hidden;
+  @include transition(all .2s $ease-in-out-quad 0s);
+
+  &.button-previous {
+    @include border-radius(35px, 0, 0, 35px);
+    left: 0;
+    background-position: center 15px;
 
-        a {
-          background-position: center 15px;
+    // CASE: left to right layout
+    @include ltr {
+      background-image: url('../images/sequence-nav/previous-icon.png');
+    }
```

Comment:
Is there a reason we are using background-images here?  I'd like to remove these from the platform whenever we can.  They are a problem for users in high-contrast mode (which removes all images on Win platform).  They don't use screen readers, so there is nothing indicating that there is a button there or what it does.  I'd like to see these in the DOM or changed over to icon fonts.